cts/Makefile.am

```diff
@@ -49,7 +49,8 @@ nodist_ctslib_PYTHON	= CTS.py		\
 ctsdir		= 	$(testdir)/cts
 cts_DATA	=	pacemaker-cts-dummyd@.service
 if BUILD_UPSTART
-dist_cts_DATA	=	pacemaker-cts-dummyd.conf
+dist_cts_DATA	=	pacemaker-cts-dummyd.conf \
+			cts.conf
```

Comment:
This block is conditioned on BUILD_UPSTART. The easiest way around that will be to put dist_cts_DATA = cts.conf first, then make pacemaker-cts-dummyd.conf use the += form.